### PR TITLE
LPS-130537 Html escape no longer needed cause blogModel title and subtitle is escaped by default so, extractText on title of vertical cards because is double escaped

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
ci:test:relevant

**Comment by liuxingye** [comment]
ci:test:relevant

File: modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/servlet/taglib/clay/BlogsEntryVerticalCard.java [hunk 1]
```diff
@@ -137,7 +137,8 @@ public String getSubtitle() {
 
 	@Override
 	public String getTitle() {
-		return BlogsEntryUtil.getDisplayTitle(_resourceBundle, _blogsEntry);
+		return HtmlUtil.extractText(
+			BlogsEntryUtil.getDisplayTitle(_resourceBundle, _blogsEntry));
```
[the remark this answers is not in the document]
Comment (adolfopa) [reply modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/servlet/taglib/clay/BlogsEntryVerticalCard.java:141]: It is fine :+1:

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
@AliciaGarciaGarcia The changes are fine, but unless this fixes a blocker, I'd prefer to wait for the release.

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**